automatic/tortoisesvn/update.ps1

```diff
@@ -0,0 +1,74 @@
+import-module au
+import-module "$PSScriptRoot/../../extensions/chocolatey-core.extension/extensions/chocolatey-core.psm1"
+
+$releases = 'https://tortoisesvn.net/downloads.html'
+
+function global:au_BeforeUpdate {
+  Remove-Item "$PSScriptRoot\tools\*.exe"
+
+  $client = New-Object System.Net.WebClient
+  try 
+  {
+    $filePath32 = "$PSScriptRoot\tools\$($Latest.FileName32)"
+    $client.DownloadFile($Latest.URL32, "$filePath32")
+
+    $filePath64 = "$PSScriptRoot\tools\$($Latest.FileName64)"
+    $client.DownloadFile($Latest.URL64, "$filePath64")
+  }
+  finally 
+  {
+    $client.Dispose()
+  }
+
+  $Latest.ChecksumType = "sha256"
+  $Latest.Checksum32 = Get-FileHash -Algorithm $Latest.ChecksumType -Path $filePath32 | ForEach-Object Hash
+  $Latest.Checksum64 = Get-FileHash -Algorithm $Latest.ChecksumType -Path $filePath64 | ForEach-Object Hash
+}
```

Comment:
hmm, possibly. I don't really know the inner workings of the .net webclient.downloadfile.
lets just keep this function as it is for now.
I'll look into it some day when the sun shines.